tasks.py

```diff
@@ -211,7 +211,21 @@ def dev_env(ctx, architecture="amd64", name="kind", cni=None):
             f.write(manifest)
             f.flush()
 
+        # Create memberlist secret.
+        secret = """apiVersion: v1
+kind: Secret
+metadata:
+  name: memberlist
+  namespace: metallb-system
+data:
+  secretkey: dmVyeXNlY3VyZWxvbAo="""
```

Comment:
he, lol. Does it make sense to use plaintext for the secret? Seems easier in case you want to debug something

Reply:
I don't see a big difference honestly. I'm fine with switching to `stringData` if you find this better.

Reply:
I slightly would prefer that, yes. Seems easier to debug and to read with the `"""` ending

Reply:
Done. I've also tried to make the YAML snippet more readable by starting the document with `---` so that the first line doesn't look out of place.

Reply:
Nice :)